storage/src/bitmap/authenticated.rs

```diff
@@ -38,7 +38,7 @@ use tracing::{debug, error, warn};
 ///
 /// Even though we use u64 identifiers for bits, on 32-bit machines, the maximum addressable bit is
 /// limited to (u32::MAX * N * 8).
-pub struct BitMap<D: Digest, const N: usize> {
+pub struct Authenticated<D: Digest, const N: usize> {
```

Comment:
Why not just keep it named Bitmap? ("authenticated" is already in the path).

Reply:
Changed back to `BitMap`